x-pack/plugins/ml/public/alerting/jobs_health_rule/register_jobs_health_alerting_rule.ts

```diff
@@ -87,24 +87,24 @@ export function registerJobsHealthAlertingRule(
     defaultActionMessage: i18n.translate(
       'xpack.ml.alertTypes.jobsHealthAlertingRule.defaultActionMessage',
       {
-        defaultMessage: `[\\{\\{rule.name\\}\\}] Anomaly detection jobs health check result:
-\\{\\{context.message\\}\\}
-\\{\\{#context.results\\}\\}
-  Job ID: \\{\\{job_id\\}\\}
-  \\{\\{#datafeed_id\\}\\}Datafeed ID: \\{\\{datafeed_id\\}\\}
-  \\{\\{/datafeed_id\\}\\}\\{\\{#datafeed_state\\}\\}Datafeed state: \\{\\{datafeed_state\\}\\}
-  \\{\\{/datafeed_state\\}\\}\\{\\{#memory_status\\}\\}Memory status: \\{\\{memory_status\\}\\}
-  \\{\\{/memory_status\\}\\}\\{\\{#model_bytes\\}\\}Model size: \\{\\{model_bytes\\}\\}
-  \\{\\{/model_bytes\\}\\}\\{\\{#model_bytes_memory_limit\\}\\}Model memory limit: \\{\\{model_bytes_memory_limit\\}\\}
-  \\{\\{/model_bytes_memory_limit\\}\\}\\{\\{#peak_model_bytes\\}\\}Peak model bytes: \\{\\{peak_model_bytes\\}\\}
-  \\{\\{/peak_model_bytes\\}\\}\\{\\{#model_bytes_exceeded\\}\\}Model exceeded: \\{\\{model_bytes_exceeded\\}\\}
-  \\{\\{/model_bytes_exceeded\\}\\}\\{\\{#log_time\\}\\}Memory logging time: \\{\\{log_time\\}\\}
-  \\{\\{/log_time\\}\\}\\{\\{#failed_category_count\\}\\}Failed category count: \\{\\{failed_category_count\\}\\}
-  \\{\\{/failed_category_count\\}\\}\\{\\{#annotation\\}\\}Annotation: \\{\\{annotation\\}\\}
-  \\{\\{/annotation\\}\\}\\{\\{#missed_docs_count\\}\\}Number of missed documents: \\{\\{missed_docs_count\\}\\}
-  \\{\\{/missed_docs_count\\}\\}\\{\\{#end_timestamp\\}\\}Latest finalized bucket with missing docs: \\{\\{end_timestamp\\}\\}
-  \\{\\{/end_timestamp\\}\\}\\{\\{#errors\\}\\}Error message: \\{\\{message\\}\\} \\{\\{/errors\\}\\}
-\\{\\{/context.results\\}\\}
+        defaultMessage: `['{{rule.name}}'] Anomaly detection jobs health check result:
+'{{context.message}}'
+'{{#context.results}}'
+  Job ID: '{{job_id}}'
+  '{{#datafeed_id}}'Datafeed ID: '{{datafeed_id}}'
+  '{{/datafeed_id}}''{{#datafeed_state}}'Datafeed state: '{{datafeed_state}}'
+  '{{/datafeed_state}}''{{#memory_status}}'Memory status: '{{memory_status}}'
+  '{{/memory_status}}''{{#model_bytes}}'Model size: '{{model_bytes}}'
+  '{{/model_bytes}}''{{#model_bytes_memory_limit}}'Model memory limit: '{{model_bytes_memory_limit}}'
+  '{{/model_bytes_memory_limit}}''{{#peak_model_bytes}}'Peak model bytes: '{{peak_model_bytes}}'
+  '{{/peak_model_bytes}}''{{#model_bytes_exceeded}}'Model exceeded: '{{model_bytes_exceeded}}'
+  '{{/model_bytes_exceeded}}''{{#log_time}}'Memory logging time: '{{log_time}}'
+  '{{/log_time}}''{{#failed_category_count}}'Failed category count: '{{failed_category_count}}'
+  '{{/failed_category_count}}''{{#annotation}}'Annotation: '{{annotation}}'
+  '{{/annotation}}''{{#missed_docs_count}}'Number of missed documents: '{{missed_docs_count}}'
+  '{{/missed_docs_count}}''{{#end_timestamp}}'Latest finalized bucket with missing docs: '{{end_timestamp}}'
+  '{{/end_timestamp}}''{{#errors}}'Error message: '{{message}}' '{{/errors}}'
+'{{/context.results}}'
```

Comment:
<img width="1245" alt="image" src="https://github.com/elastic/kibana/assets/5236598/cacff0bf-c002-45d2-9d18-432e82e07c54">

Something wrong with the quotes here

Reply:
fixed. please take a look